packages/apps/plugins/plugin-layout/src/components/MainLayout.tsx

```diff
@@ -82,7 +82,7 @@ export const MainLayout = ({ fullscreen, showComplementarySidebar = true }: Main
                   <MenuIcon weight='light' className={getSize(4)} />
                 </Button>
 
-                <Surface role='heading' limit={2} />
+                <Surface role='navbar-start' limit={1} />
```

Comment:
why limit 1? if it's limit one and used by a core plugin it's essentially not extensible

Reply:
It’s a horizontal layout, which I don’t think plugins can coordinate. I’d argue it’s not extensible by the design provided by LayoutPlugin, since it does not accommodate overflow.

If it should be unlimited, I think we basically need to redesign this surface.